pkg/resources/tag_masking_policy_association_test.go

```diff
@@ -1,94 +0,0 @@
-package resources_test
-
-import (
-	"database/sql"
-	"regexp"
-	"testing"
-
-	sqlmock "github.com/DATA-DOG/go-sqlmock"
-	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/provider"
-	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/resources"
-
-	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/snowflake"
-	. "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/testhelpers"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/stretchr/testify/require"
-)
-
-func TestTagMaskingPolicyAssociation(t *testing.T) {
-	r := require.New(t)
-	err := resources.Tag().InternalValidate(provider.Provider().Schema, true)
-	r.NoError(err)
-}
```

Comment:
No more tests?

Reply:
these are not very useful tests, and to get them to work i would have to add in the commands for creating a temporary warehouse.